app/models/primer.rb

```diff
@@ -5,9 +5,21 @@ class Primer < ActiveRecord::Base
   has_many :marker_assays_b, class_name: 'MarkerAssay',
              foreign_key: 'primer_b_id'
 
+  include Annotable
```

Comment:
Why did you move this `include` here? There is an important reason why this is included in the last line of the model definition.